src/Shared/Shared.fs

```diff
@@ -46,10 +46,12 @@ module SorensenDice =
 
 type IOntologyAPIv3 = {
     // Development
-    getTestNumber : unit -> Async<int>
-    searchTerm: TermQuery -> Async<Term []>
-    searchTerms: TermQuery[] -> Async<TermQueryResults[]>
-    getTermById: string -> Async<Term option>
+    getTestNumber           : unit          -> Async<int>
+    searchTerm              : TermQuery     -> Async<Term []>
+    searchTerms             : TermQuery[]   -> Async<TermQueryResults[]>
+    getTermById             : string        -> Async<Term option>
+    findAllChildTerms       : TermQuery     -> Async<Term []>
```

Comment:
Do not use TermQuery. Introduce a new type, tailored to the API request. You are not using like 50% of the type.

Reply:
fixed